### DISCUSSION TITLE
Resaltador de código con botón para copiar

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#25] [Announcements]
# Resaltador de código con botón para copiar

Resaltador de código optimizado usando highlight.js que incluye el botón para copiar

https://www.zkreations.com/2017/05/utilizar-highlightjs-para-resaltar.html

**Comment by ghost** [comment]
Hola, que tendria que modificar si solo es texto plano? gracias.

**Reply by danieIabel** [reply]
Buenas noches amigo. Si solo es texto plano no tienes que hacer nada, solo es necesario escapar el codigo si tiene html
